### ISSUE TITLE
Review colors for accessibility contrast failures

### ISSUE THREAD

**Issue by dotNomad** [posit-dev/connect-extensions#220]
There are a few low contrast bits of text in the Package Vulnerability Scanner. These should be fixed to meet accessibility standards.
